packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -69,19 +75,19 @@ export type FormFieldConfig<T> = (
           labelField?: string;
           filterField?: { name: string; gqlName?: string };
       } & Omit<InputBaseFieldConfig, "endAdornment">)
-    | { type: "block"; block: ImportReference }
+    | { type: "block"; block: BlockInterface }
     | SingleFileFormFieldConfig
     | MultiFileFormFieldConfig
 ) & {
     name: keyof T;
     label?: string;
     required?: boolean;
     virtual?: boolean;
-    validate?: ImportReference;
+    validate?: (value: unknown) => ReactNode | undefined;
```

Comment:
Do we still support imports here? Might come in handy if we want to reuse the same validation for multiple forms.

Also, you can use the `FieldValidator` type here, which includes allValues and meta in the function declaration.

Reply:
> Do we still support imports here?

yes, we do. It's just not visible anymore here in the types.